README.md

```diff
@@ -35,9 +35,10 @@ The module can be loaded to Linux kernel by runnning the command:
 $ sudo insmod vcam.ko
 ```
 
-Expectedly, two device nodes will be created in `/dev`:
+Expectedly, three device nodes will be created in `/dev`:
 * videoX - V4L2 device;
 * vcamctl - Control device for virtual camera(s), used by control utility `vcam-util`;
+* fbX - frame buffer device;
```

Comment:
Change the description to
> controlling framebuffer device

No space is separated between "frame" and "buffer."

Reply:
See https://github.com/dvdhrm/docs/blob/master/drm-howto/modeset.c